### PR TITLE
fix(Makefile): print error if pip is missing

### LINKED ISSUE #362: `make bootstrap` should test for pip and then run pip install
Right now, it's possible to run through the developer setup and still have a broken set of tests because the PIP install for expandybird is not automated. This also leads to redundant steps in Dockerfiles and the CircleCI build.

### PR DESCRIPTION
**Pull request by technosophos**
Also, prompt the user to run pip install during bootstrap.

Closes #362.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -101,3 +105,4 @@ ifndef HAS_GOX
 	go get -u github.com/mitchellh/gox
 endif
 	glide install
+	@echo Please run pip install -r expansion/requirements.txt. You might need sudo.
```
Comment (adamreese) [inline Makefile]: Is there a reason for not running `pip install ...` here?

Comment (technosophos) [inline Makefile]: Reason is that some systems require sudo and others do not.

Comment (mboersma) [inline Makefile]: `pip install --user -r expansion/requirements.txt` might be what you're looking for.

```
  --user                      Install to the Python user install directory for your platform. Typically ~/.local/, or %APPDATA%\Python on Windows. (See the Python
                              documentation for site.USER_BASE for full details.)
```

Comment (technosophos) [inline Makefile]: Will this muck with virtual environments or anything, @mboersma ? Any chance that this will be a nasty surprise to the user?

Comment (mboersma) [inline Makefile]: > Will this muck with virtual environments...?

It apparently just won't work:
```console
$ . venv/bin/activate
(venv) $ pip install --user django
Can not perform a '--user' install. User site-packages are not visible in this virtualenv.
```

**Comment by jackgr** [comment]
@technosophos: Is this PR awaiting review or are there open issues to be resolved first?

**Pull request merged**